bootcommand/usb_driver.go

```diff
@@ -55,8 +55,8 @@ func NewUSBDriver(send SendUsbScanCodes, interval time.Duration) *usbDriver {
 		"insert":     key.CodeInsert,
 		"home":       key.CodeHome,
 		"end":        key.CodeEnd,
-		"pageUp":     key.CodePageUp,
-		"pageDown":   key.CodePageDown,
```

Comment:
Does that mean that `pageUp` will stop working ?

Reply:
They never worked. The keystrokes are lowercased before matching the map. All of the others are lowercased I don't know why I added these two camel-cased in the first place. 🤔